applications/nrf5340_audio/src/bluetooth/ble_audio_services.c

```diff
@@ -336,184 +321,194 @@ static void mcs_local_player_instance_cb(struct media_player *player, int err)
 	 */
 	(void)media_proxy_ctrl_send_command(local_player, &cmd);
 }
-#endif /* (CONFIG_BT_MCS) */
 
 int ble_vcs_vol_set(uint8_t volume)
 {
-#if (CONFIG_BT_VCP_VOL_CTLR)
-	int ret;
+	if (IS_ENABLED(CONFIG_BT_VCP_VOL_CTLR)) {
+		int ret;
 
-	for (int i = 0; i < CONFIG_BT_MAX_CONN; i++) {
-		ret = ble_vcs_client_remote_set(i);
-		/* If remote peer hasn't been connected before, just skip the operation for it */
-		if (ret == -EINVAL) {
-			continue;
-		}
-		ERR_CHK_MSG(ret, "Failed to set VCS client to remote device properly");
-		ret = bt_vcp_vol_ctlr_set_vol(vcs, volume);
-		if (ret) {
-			LOG_WRN("Failed to set volume for remote channel %d, ret = %d", i, ret);
+		for (int i = 0; i < CONFIG_BT_MAX_CONN; i++) {
+			ret = ble_vcs_client_remote_set(i);
+			/* If remote peer hasn't been connected before, just skip the operation */
+			if (ret == -EINVAL) {
+				continue;
+			}
+			ERR_CHK_MSG(ret, "Failed to set VCS client to remote device properly");
+			ret = bt_vcp_vol_ctlr_set_vol(vcs, volume);
+			if (ret) {
+				LOG_WRN("Failed to set volume for remote channel %d, ret = %d", i,
+					ret);
+			}
 		}
+		return 0;
+	} else if (IS_ENABLED(CONFIG_BT_VCP_VOL_REND)) {
+		return bt_vcp_vol_rend_set_vol(volume);
+	} else {
+		return hw_codec_volume_set(vcs_vol_conversion(volume));
 	}
```

Comment:
Very nice PR, just one minor comment: you don't need to have an else since there is a return in the each of the ifs above. This goes for all of these functions related to volume

```suggestion
	}
	
	return hw_codec_volume_set(vcs_vol_conversion(volume));
```

Reply:
Maybe add a LOG_DBG("Volume control service not enabled, setting volume locally"); or something similar

Reply:
Added LOG_DBG for all the instances